Sources/Slacket/Adapters/BodyParser.swift

```diff
@@ -0,0 +1,117 @@
+//
```

Comment:
The filename is `BodyParser` which was renamed from `ParserType` and eventually the file contains a `ParsedBody` extension. The filename collides with kitura's `BodyParser` while were at that.